### ISSUE TITLE
Need help on: 

### ISSUE THREAD

**Issue by BenPilliez** [diegoazh/gmap-vue#232]
> Please complete all the sections that you can but please **don't remove any section**, otherwise the bot will close your issue because it doesn't follow our issue template (you can remove this comment)

## What you try to do

Hi i'm trying do implement a custom v-text-field in the autocomplete component

## What is the problem

My custom text-field is not displaying correctly, it just ignoring

## What kind of things have you already treated
`<gmap-autocomplete @place_changed="changeCity" class="gmAutocomplete">
        <template v-slot:input="slotProps">
          <v-text-field
            ref="input"
            name="input"
            v-model="form.country.label"
            :placeholder="$t('selectCity.chooseCity')"
            append-icon="mdi-map-marker"
            outlined
            dense
            clearable
            v-on:attrs="slotProps.attrs"
            v-on:listeners="slotProps.listeners">
          </v-text-field>
        </template>
      </gmap-autocomplete>`

## Steps to reproduce

Explain to us how to get the same problem or provide a small example repository to work on it.

## Os

- [ ] Mac Os X
- [ ] Linux (write here the name and version of your distribution)
- [x ] Windows

## Mobile

- [ ] iOS
- [ ] Android

## Versions

- Node: 10.18.0
- NPM: 6.13.4
- Yarn:

## Package manager

- [ ] NPM
- [ ] Yarn

## Plugin version
- [x] 2.0.x
- [ ] 1.4.x
- [ ] 1.2.x
- [ ] 1.0.x

**Comment by diegoazh** [comment]
Hi @BenPilliez can you provide a minimal reproduction repository?

**Comment by Mario996** [comment]
Hello @diegoazh, I just stumbled upon this since I had the same issue as OP, but I resolved it thanks to your comment. Maybe the documentation should be updated to be aligned with the newest version and then you display a warning like you did on top of the page stating about that slot change. It makes more sense for people with latest version to be able to use code from documentation right away. Anyhow thank you for your contributions to this great plugin.
